packages/nimble-components/src/theme-provider/index.ts

```diff
@@ -43,6 +53,17 @@ export class ThemeProvider extends FoundationElement {
     })
     public theme: Theme = Theme.light;
 
+    public langChanged(
+        _prev: string | undefined,
+        next: string | undefined
+    ): void {
+        if (next !== undefined && next !== null) {
```

Comment:
It's interesting that we're checking for `null` here when that isn't called out as an allowed value. Any idea why? Looks like that's also the case for `directionChanged` and `themeChanged`.

Reply:
I'm not sure if the property could reasonably end up with the value `null`. I assume there is a way, since the other property change handlers are written the same way.

Reply:
Calling `element.removeAttribute(attributeName)` will cause the `Changed()` methods to be called with `next` as `null` (similarly, going from no attribute set to another value yields `_prev` as `null`)

Reply:
I updated the signatures as suggested.